src/lib/plugins/sast/analysis.ts

```
@@ -133,3 +138,11 @@ function getSecurityResultsOnly(
 
   return securityResults;
 }
+
+function isLocalCodeEngine(sastSettings: SastSettings): boolean {
+  const { sastEnabled, localCodeEngine } = sastSettings;
+
+  return (
+    sastEnabled && localCodeEngine.enabled && localCodeEngine.url.length > 0
```

if `localCodeEngine.enabled` is true and  `localCodeEngine.url.length` is empty, which is possible to happen, that the client has misconfigured the settings.

In such case, we want to throw an error, saying something like: 
```
You have Snyk Code Local Engine enabled, but no URL is provided.
Check our docs to learn more (documentation for this is still to be created, we need to remember to insert the link here when we have it)
```

But for sure we do not want to consider `isLocalCodeEngine` === false if localCodeEngine.url.length === 0